index.html

```diff
@@ -456,32 +421,171 @@ <h2 id="constraints">
+              <td><code><a>ConstrainDouble</a></code></td>
+              <td>The frame rate (frames per second) or frame rate range. As a
+              capability, max MUST reflect the <a>display surface</a>'s frame
+              rate, and min MUST reflect the lowest frame rate available through
+              frame decimation by the user agent.</td>
+            </tr>
+            <tr id="def-constraint-aspect">
+              <td><dfn>aspectRatio</dfn></td>
+              <td><code><a>ConstrainDouble</a></code></td>
+              <td>The exact aspect ratio (width in pixels divided by height in
+              pixels, represented as a double rounded to the tenth decimal
+              place) or aspect ratio range. As a setting, represents the aspect
+              ratio of the <a>display surface</a>. As a capability, min and max
+              both MUST be the setting value, rendering this property immutable
+              from the application viewpoint. However, these values may change
+              if the end-user is able to resize the <a>display surface</a>.</td>
+            </tr>
+            <tr id="def-constraint-resizeMode">
+              <td><dfn>resizeMode</dfn></td>
+              <td><code><a>ConstrainDOMString</a></code></td>
+              <td>
+                This string (or each string, when a list) should be one of the
+                members of <code><a>VideoResizeModeEnum</a></code>. As a setting,
+                <code>none</code> means the <code><a>MediaStreamTrack</a></code>
+                contains all bits needed to render the display in full detail,
+                which if <code>window.devicePixelRatio > 1</code>, means
+                <code>width</code> and <code>height</code> will be larger
+                than the display's appearance from an end-user viewpoint would
+                suggest, whereas <code>crop-and-scale</code> means the
+                <code><a>MediaStreamTrack</a></code> contains an
+                aspect-preserved representation of the <a>display surface</a>
+                that has been downscaled by the user agent, but not cropped. As
+                a capability, the values <code>none</code> and
+                <code>crop-and-scale</code> both MUST be present. In other
+                words, the user agent MUST support downscaling.
+              </td>
+            </tr>
+            <tr id="def-constraint-displaySurface">
+              <td><dfn>displaySurface</dfn></td>
+              <td><code><a>ConstrainDOMString</a></code></td>
+              <td>
+                This string (or each string, when a list) should be one of the
+                members of <code><a>DisplayCaptureSurfaceType</a></code>. As a
+                setting, indicates the type of <a>display surface</a> that
+                is being captured. As a capability, the setting value MUST be
+                the lone value present, rendering this property immutable from
```

Comment:
What does "the lone value present" mean? The string "present"?

Reply:
As a capability, the type is `sequence<DOMString>`. I'll add a PR to expand [MediaTrackCapabilities](https://w3c.github.io/mediacapture-main/getusermedia.html#media-track-capabilities).